io.github.pantheon_tweaks.pantheon-tweaks.yml

```diff
@@ -0,0 +1,112 @@
+app-id: io.github.pantheon_tweaks.pantheon-tweaks
+# elementary SDK is not available on Flathub, so use GNOME SDK and install elementary stylesheet and granite as modules
+runtime: org.gnome.Platform
+runtime-version: '46'
+sdk: org.gnome.Sdk
+command: start-pantheon-tweaks
+finish-args:
+  - '--share=ipc'
+  - '--socket=wayland'
+  - '--socket=fallback-x11'
+
+  # Grant read, write, and create access for settings.ini of GTK and theme dirs
+  - '--filesystem=home/.config/gtk-3.0:create'
+  - '--filesystem=home/.icons:create'
+  - '--filesystem=home/.local/share/sounds:create'
+  - '--filesystem=home/.local/share/themes:create'
```

Comment:
these broad permission aren't what you think they are.
`home` doesn't have a specific mean in that context.

Reply:
Did you mean, for example, `--filesystem=home/.local/share/themes:create` would NOT give us create access to the host's `~/.local/share/themes` directory?

I confirmed the app could create these directories if these lines exist and could not if I delete these lines, at least on my installation of elementary OS 7.1 (Flatpak 1.12.7 + Flatpak Builder from Flathub).

Reply:
Fixed in b507a85e8f7739c8eadaae21d73ee7b84142a512